src/quickgui/plugin/qgsquickmapcanvas.qml

```diff
@@ -114,6 +137,40 @@ Item {
     freeze: false
   }
 
+  Item {
+    id: jumpAnimator
+
+    property double px: 0
+    property double py: 0
+
+    Behavior on py {
+      SmoothedAnimation {
+        duration: 200
+        reversingMode: SmoothedAnimation.Immediate
+      }
+      enabled: jumpAnimator.enabled
+    }
+
+    Behavior on px {
+      SmoothedAnimation {
+        duration: 200
+        reversingMode: SmoothedAnimation.Immediate
+      }
+      enabled: jumpAnimator.enabled
+    }
+
+    onPxChanged: {
+      if ( enabled ) {
+        mapCanvasWrapper.mapSettings.setCenter( utils.toQgsPoint( Qt.point( px, py ) ) )
```

Comment:
```suggestion
        mapCanvasWrapper.mapSettings.center = utils.toQgsPoint( Qt.point( px, py ) )
```

Since center is a property, that would look a bit more QML'esque

Reply:
Good point